### DISCUSSION TITLE
@mdx-js/rollup plugin doesn't work in development mode

### DISCUSSION THREAD

**Discussion by daron81** [preactjs/wmr#925] [Q&A]
Hello,

I'm trying to use the @mdx-js/rollup plugin with wmr, however it errors out when running in development mode with the following message: Cannot read properties of undefined (reading '0').

When building with wmr --prerender, the md is correctly rendered to html.

Any ideas on what is going on?

**Comment by rschristian** [comment]
It's a problem with the sourcemap merging, I'm working on fix now. Going to convert this into an issue.

**Reply by rschristian** [reply]
#926

**Comment by daron81** [comment]
Thanks for that! I'll follow the issue.
